docs/source/sft_trainer.mdx

```diff
@@ -594,6 +594,29 @@ trainer.train()
 
 The saved model is fully compatible with Hugging Face's transformers library. Learn more about unsloth in their [official repository](https://github.com/unslothai/unsloth).
 
+### Increase 20% throughput and reduces 60% memory using Liger-Kernel
+
+[Liger Kernel](https://github.com/linkedin/Liger-Kernel) is a collection of Triton kernels designed specifically for LLM training. It can effectively increase multi-GPU training throughput by 20% and reduces memory usage by 60%. That way, we can **4x** our context length, as described in the benchmark below. They have implemented Hugging Face Compatible `RMSNorm`, `RoPE`, `SwiGLU`, `CrossEntropy`, `FusedLinearCrossEntropy`, and more to come.
+
+| Speed Up                 | Memory Reduction        |
+|--------------------------|-------------------------|
+| ![Speed up](https://raw.githubusercontent.com/linkedin/Liger-Kernel/main/docs/images/e2e-tps.png) | ![Memory](https://raw.githubusercontent.com/linkedin/Liger-Kernel/main/docs/images/e2e-memory.png) |
+
+To use Liger-Kernel in `SFTTrainer`, you can first install Liger-Kernel according to [the official website](https://github.com/linkedin/Liger-Kernel). Once installed, we can use it out-of-the-box with two simple line changes:
```

Comment:
```suggestion

1. To use Liger-Kernel in `SFTTrainer`, first install by 

```bash
pip install liger-kernel
```

2. Once installed, set `use_liger` in [SFTConfig](https://github.com/huggingface/trl/blob/850ddcf598984013007d384c6b3e311def2a616e/trl/trainer/sft_config.py#L69). No other changes are needed!


```python

config = SFTConfig(
  use_liger=True
)
```
```

Reply:
the format of suggestion is broken